cmd/plugins/topology-aware/policy/pod-preferences.go

```diff
@@ -44,6 +44,8 @@ const (
 	keyReservedCPUsPreference = "prefer-reserved-cpus"
 	// annotation key for CPU Priority preference
 	keyCpuPriorityPreference = "prefer-cpu-priority"
+	// annotation key for hiding hyperthreads from allocated CPU sets
+	keyHideHyperthreads = "hide-hyperthreads"
```

Comment:
Do/don't we have something like this in the balloons policy, and if we do, is this key consistent with the annotation used there ?

Reply:
Balloons is still in my feature branch and it does not support annotation at all. Now that we have topology-aware implementation in place, I know how to rewrite the balloons part so that it'll fit in the picture. (Add support for the same annotation and start using the SingleThread filter from system.go instead of having it in cputree.go)

Reply:
Couldn't really find a good place for a common annotation. Still thinking if we should have this annotation parser in the cache package, similarly to preserve-* annotations. Or will I use only GetEffectiveAnnotation from there and parse the boolean in balloons and topology-aware separately - which is the direction that this PR is now trending to. What do you think @klihub? I can still move it to the cache interface, but on the other hand, I would not like to add there too many annotations that may not make sense in every policy.

Reply:
I think putting it to `pkg/resmgr/cache` implies that we expect all policies to implement it, so we should be careful with that. Although is a simple boolean preference which is semantically simple enough that it might fit that bill, I have no strong opinions and am fine with either way.